ap.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Reads for User Input
+echo "Because of vulnerabilities with other security, we will use WPA2"
+echo "What Name Do You Want The WiFi To Have?:"
+read ssid
+echo "What Password Should It Have?:"
+read key
+echo "What Channel?: (To Avoid Congestion, Use A Channel Not Popular In The Area, ie: 3,5,7,9)"
+read channel
+# Launch Time
+sudo create_ap -w 2 -c $channel $wifdev eth0 $ssid $key
```

Comment:
Also not on all systems eth0 is the desired ethernet controller. For example on my system it is named `enp0s25`. And that is the reason why program parameters exist. If you want to run the same command over and over again I suggest creating a proper config, as it is intended or use the history function of bash to reload the command.